docs/Contratos.html

```diff
@@ -0,0 +1,113 @@
+	  href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" 
+	  integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" 
+	  crossorigin="anonymous">
+	  
+    <title>Contratos</title>
+  </head>
+  <body>
+       <!--fixed size -->
+    <div class= "container">
+        <h4><b>Capturar contrato</b></h4>
+        <br>
+        <h6 class="text-secondary">Datos de la Escuela</h6>
+        <form>
+            <!--row 1-->
+            <div class="form-row">
+                <div class="form-group col-md-3">
+                    <label for="clave-escolar">Clave escolar</label>
+                    <input type="text" class="form-control" name="clave-escolar" id="clave-escolar"
+                    placeholder="Clave escolar">
+                </div>
+                <div class="form-group col-md-7">
+                    <label for="Nombre-de-la-escuela">Nombre de la escuela</label>
+                    <input type="text" class="form-control" name= "Nombre-de-la-escuela" id="Nombre-de-la-escuela"
+                    placeholder="Nombre de la escuela">
+                </div>
+                <div class="form-group col-md-2 mt-2">
+                    <input class="btn btn-primary mt-4" type="button" value="Buscar">
+                </div>
+            </div>
+            <br>
+            <h6 class="text-secondary">Datos del contrato</h6>
+            <!--row2-->
+            <div class="form-row">
+                <div class="form-group col-md-4" >
+                    <label for="Fecha-inicio">Fecha inicio</label>
+                    <input type="date" class="form-control" name="Fecha-inicio" id="Fecha-inicio">
+                </div>
+                <div class="form-group col-md-4">
+                    <label for="Fecha-fin">Fecha fin</label>
+                    <input type="date" class="form-control" name="Fecha-fin" id="Fecha-fin">
+                </div>
+                <div class="form group col-md-4">
+                    <label for="Escuela-asociada">Escuela asociada</label>
+                    <input type="text" class="form-control" name="Escuela-asociada" id="Escuela-asociada" 
+                    placeholder="Escuela seleccionada anteriormente" disabled>
+                </div>
+            </div>
+            <!--row3-->
+            <div class="form-row">
+                <div class="form-group col-md-7">
+                    <label for="Responsable-del-contrato">Responsable del contrato</label>
+                    <input type="text" class="form-control" name="Responsable de la escuela" 
+                    id="Responsble-de-la-escuela" placeholder="Responsable de la escuela">
+                </div>
+                <div class="form group col-md-5">
+                    <label for="Doc-pdf">Documento PDF con firmas</label>
+                    <div class="input-group">
+                        <input type="file" class="custom-file-input" id="validatedCustomFile" required>
```

Comment:
El atributo `for` de una etiqueta label debe hacer referencia al `name` de un input, en este caso te hace falta la referencia de ese for, ya que no hay ningún elemento con `name="Doc-pdf"`. Agregarlo en el Input, respetando camelCase.

Reply:
Sin embargo, lo que sucede es que está anidado otro div abajo, por ello el *label* de abajo respeta la referencia al *id* del *input* subsecuente:
`<div class="form group col-md-5">
					<label for="DocPdf">Documento PDF con firmas</label>
					<div class="input-group">
						<input type="file" class="custom-file-input" id="ValidatedCustomFile" required>
						<label class="custom-file-label" for="ValidatedCustomFile">Seleccionar archivo</label>
					</div>
				</div>`

Reply:
Cierto, no vi el label de abajo, entonces lo que habría que hacer es eliminar el for del label de arriba, dado que no está siendo asociado a ningún name.